src/session.rs

```diff
@@ -120,6 +117,11 @@ pub fn start(config: &Config, initial_request: Option<String>) -> i32 {
                         // before. In that case didClose can be safely ignored as well.
                         if request.method != notification::DidCloseTextDocument::METHOD {
                             debug!("Spawning a new controller for {:?}", route);
+                            return_request_error(
+                                editor.to_editor.sender(),
+                                &request,
+                                "Starting language server"
```

Comment:
I usually run `lsp-enable` via a key binding. Whenever I do that, this shows "Starting language server" in the info box.
This info box is also shown when running a request (like `lsp-hover`) after the language server has exited (could be after a timeout).

Not sure how I feel about these, they seem a bit redundant.

It's not really an error; most requests should proceed as normal.
Only blocking requests can't be fulfilled, due to technical limitations - hence the `cancelling blocking request` message.

Reply:
Yeah, it's probably too noisy. I think it should be possible to collapse this into the "no longer running" case anyways so that a language server dying only results on one failed request and a regular successful start is always silent.

Reply:
Yeah let's try to do that. Our default timeout of 30 minutes means this may happen a few times in a working day, so it's worth improving